functions/go/fix/v1alpha2/types.go

```diff
@@ -0,0 +1,281 @@
+package v1alpha2
```

Comment:
This looks like the whole file is copied. Why not pull it in as a dependency?

Reply:
There are 2 caveats to import it as dependency
1. We should import 2 versions of kpt which increases image size.
2. kyaml transitive dependency conflicts arises. 

I think it is decent to copy and lock the types from both versions of kpt.